packages/lesswrong/server/vulcan-lib/apollo-ssr/pageCache.ts

```diff
@@ -187,6 +210,34 @@ const cacheStore = (cacheKey: string, abTestGroups: RelevantTestGroupAllocation,
     cachedABtestsIndex[cacheKey] = [abTestGroups];
 }
 
+const cacheStoreDB = (cacheKey: string, abTestGroups: RelevantTestGroupAllocation, rendered: RenderResult): void => {
+  const { bundleHash } = getClientBundle();
+
+  void PageCache.rawUpdateOne(
```

Comment:
Is there a primary index on anything? Won't this always be an insert?

Reply:
There's a unique index on `path, abTestGroups, bundleHash` and it can be an update if two servers start rendering an uncached page around the same time, and then both try to insert it